src/main/java/org/json/HTTP.java

```diff
@@ -113,24 +113,38 @@ public static JSONObject toJSONObject(String string) throws JSONException {
     public static String toString(JSONObject jo) throws JSONException {
         StringBuilder       sb = new StringBuilder();
         if (jo.has("Status-Code") && jo.has("Reason-Phrase")) {
-            sb.append(jo.getString("HTTP-Version"));
```

Comment:
I am not sure that the refactoring update in this method is a good idea. Although it shortens the method, it hides a more interesting potential refactoring - notice how many times `status-code`, `reason-phrase`, `method`, `request-uri`, and `http-version` are referenced in the conditionals. It makes the code look half-finished. Can you come up with a way to reduce the number of conditions being checked without affecting the output ordering?